docs/usage.rst

```diff
@@ -74,6 +76,39 @@ with the fake file system functions and modules:
 The usage is explained in more detail in :ref:`auto_patch` and
 demonstrated in the files `example.py`_ and `example_test.py`_.
 
+If your setup is the same for all tests in a class, you can use the class setup
+method ``setUpClassPyfakefs`` instead:
+
+.. code:: python
+
+    from pyfakefs.fake_filesystem_unittest import TestCase
+
+
+    class ExampleTestCase(TestCase):
+        def setUpClass(cls):
+            self.setUpClassPyfakefs()
+            cls.file_path = "/test/file.txt"
```

Comment:
Why so complicated? ;)
Just do

`pathlib.Path('/test/file.txt').touch()`

Isn't one of the great things of your package that I can do all file operations as usual because your package does the patching in background for me. No need to use `fake_fs()` or `create_file()`. Or do I misunderstand something?

Reply:
No, you understand correctly, this is just to demonstrate that you _can_ access the fake fs from there, say to setup things that cannot be done in the real fs (like setting the file system size). But I may change this to make this more clear.

Reply:
I should have waited for a review before merging this - I tend to overlook simple stuff like that myself, especially in the evenings after my day job... next time.